huaweicloud/resource_huaweicloud_identity_group_membership.go

```diff
@@ -147,16 +153,24 @@ func resourceIdentityGroupMembershipV3Delete(d *schema.ResourceData, meta interf
 func addUsersToGroup(identityClient *golangsdk.ServiceClient, group string, userList []string) error {
 	for _, u := range userList {
 		if r := users.AddToGroup(identityClient, group, u).ExtractErr(); r != nil {
-			return fmtp.Errorf("Error add user %s to group %s: %s ", group, u, r)
+			return fmtp.Errorf("Error add user %s to group %s: %s ", u, group, r)
 		}
 	}
 	return nil
 }
 
-func removeUsersFromGroup(identityClient *golangsdk.ServiceClient, group string, userList []string) error {
+func removeUsersFromGroup(identityClient, iamClient *golangsdk.ServiceClient, group string, userList []string) error {
 	for _, u := range userList {
+		_, err := iamUsers.Get(iamClient, u).Extract()
```

Comment:
we can also get user details with identityClient, please see [here](https://support.huaweicloud.com/intl/en-us/api-iam/iam_08_0002.html).

Reply:
found a better way, please have a look